src/controller/python/chip-device-ctrl.py

```diff
@@ -762,6 +762,40 @@ def resolve(fabric_id: int, node_id: int) -> Dict[str, Any]:
     except Exception as e:
         return __get_response_dict(status = StatusCodeEnum.FAILED, error = str(e))
 
+def zcl_command(cluster: str,command: str, node_id: int, endpoint_id: Optional[int] = 1, group_id: Optional[int] = 0, args: Optional[dict] = {}) -> Dict[str, Any] :
+    try:
+        __check_supported_os()
+
+        all_commands = device_manager.devCtrl.ZCLCommandList()
+        command_arg = all_commands.get(cluster).get(command, None)
+        if command_arg is None:
+            raise exceptions.UnknownCommand(cluster, command)
+        commandArgs = __format_zcl_arguments_from_dict(args, command_arg)
```

Comment:
please use snake case for `commandArgs` since it is just a variable. Camel case should only be used by class definition names if the rest of the file adheres to that.

Since you already have a command_arg consider renaming `command_args` to something like zcl_arguments based on the method you are calling to fill it.

Reply:
Ack